src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -0,0 +1,62 @@
+---
+title: Using different type of storage 
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/using-diffrent-type-of-storage/
+folder: che-7/end-user-guide
+summary:
+---
+
+[id="using-diffrent-type-of-storage_{context}"]
+= Using different type of storage 
+
+Since 7.17.0 Eclipse Che supports three types of storage with different capabilities: persistent storage, ephemeral and asynchronous.
+
+.Persistent storage
+
+This kind of storage gives you an ability to store your changes directly in the mounted Persistent Volume. In this case your changes will be totally safe and Kubernetes infrastructure takes care of it (storage backend in particular), but the price for that is slow I/O, especially with a huge amount of small files. For example, some NodeJS projects may have many dependencies and node_modules is filled with thousands of small files.
```

Comment:
maybe we should explain that I/O depends on the storage backend that is used, not just that is slow. Probably provide a comparison of I/O against EBS and against emptyDir

Reply:
done